pkg/controller/ipam/antrea_ipam_controller.go

```diff
@@ -287,7 +305,11 @@ func (c *AntreaIPAMController) getIPPoolsForStatefulSet(ss *appsv1.StatefulSet)
 func (c *AntreaIPAMController) preallocateIPPoolForStatefulSet(ss *appsv1.StatefulSet) error {
 	klog.InfoS("Processing create notification", "Namespace", ss.Namespace, "StatefulSet", ss.Name)
 
-	ipPools := c.getIPPoolsForStatefulSet(ss)
+	ipPools, ips := c.getIPPoolsForStatefulSet(ss)
+	var ip net.IP
+	if len(ips) > 0 {
+		ip = ips[0]
+	}
```

Comment:
I didn't get this, why only the first one is used? how about the case that size of replicas are larger than 1?

Reply:
We only support 1 IP here currently, and we require replicas=1 for StatesulSet (Documented in Antrea Doc)
We might support FlexibleIPAM with IPv6/DS in future, thus the ip annotation is a list.